tempodb/tempodb.go

```diff
@@ -319,11 +326,11 @@ func (rw *readerWriter) Find(ctx context.Context, tenantID string, id encoding.I
 				break
 			}
 		}
-		level.Info(logger).Log("msg", "searching for trace in block", "traceID", hex.EncodeToString(id), "block", meta.BlockID, "found", foundObject != nil)
-		span.LogFields(ot_log.String("msg", "searching for trace in block"), ot_log.String("traceID", hex.EncodeToString(id)), ot_log.String("block", meta.BlockID.String()), ot_log.Bool("found", foundObject != nil))
-		if foundObject != nil {
-			span.SetTag("object bytes", len(foundObject))
-		}
+		level.Info(logger).Log("msg", "searching for trace in block", "findTraceID", hex.EncodeToString(id), "block", meta.BlockID, "found", foundObject != nil)
+		span.LogFields(ot_log.String("msg", "complete"),
+			ot_log.String("findTraceID", hex.EncodeToString(id)),
+			ot_log.Bool("found", foundObject != nil),
+			ot_log.Int("bytes", len(foundObject)))
```

Comment:
Moving this log line to the `block.Find` span is a little concerning to me. Previously I was able to look at the `store.Find` for a trace and check which blocks were queried and whether the trace was found in each, but with this change we'd have to go through each of the `block.Find`s. Can we add one final log line after this `RunJobs` call to log the block with the successful find?

Reply:
Agree that this is annoying in the new traces.  I kind of want to just drop all the memcached spans somehow, but that would be super ugly.

I'm continuing work on this today and will see if there's something I can do to cleanup.

Reply:
Cool, if you're continuing on this its fine. We'll have a better idea looking at the new traces as well :)